### DISCUSSION TITLE
Preferences backup issue

### DISCUSSION THREAD

**Discussion by davidovic999** [mikeberger/borg_calendar#207] [General]
Hello Mike,
When you run the backup, it appears that the preferences are not saved.
All preference settings: font family, font size, theme, etc...
Because when I imported the backup zip file, these settings were not imported!

**Comment by mikeberger** [comment]
Preferences are not in the database, they are stored using the Java preference system, which uses the registry on Windows and a hidden directory tree on Linux. The options menu contains entries to export and import preferences.

Preferences are machine specific. If you import the backup on a difference machine, some preferences, like the fonts, database location, look and feel - might not work and might cause problems, especially between windows and linux.

I keep my database on a shared cloud drive and run BORG on multiple computers. If the preferences were in the database, they'd be duplicated across machines. So, they are kept outside the database and are not part of the database backup.

**Comment by davidovic999** [comment]
Makes sense. Thanks for telling me where to access it.
